problem-3/server/public/index.php

```diff
@@ -0,0 +1,27 @@
+<? php
+
+require '../vendor/autoload.php';  
+ 
+use Teamleader/CodingTest/Logger;
+$logger = new Logger();
+
+// this is an example php file, so that the the project structure is clear to candidates
+
+// you may assume this code connects to some database with the following credentials
+
+$database_host = "127.0.0.1";
```

Comment:
I've put it in intentionally... same about the above
I was going to add something to the exercise : advise the team lead about how to improve the application for running it containerized